### PR TITLE
Add voice broadcast labs setting

### PR DESCRIPTION
**Pull request by Florian14**
## Type of change

- [x] Feature
- [ ] Bugfix
- [ ] Technical
- [ ] Other :

## Content

Remove the debug feature flag and move it to labs setting
Also renamed the recorded voice message files
Also changed the default voice messages duration 

Note that the flag is not accessible on Android < 10 because the recording is not possible on these versions.
The flag only forbid voice broadcast recording but the listening is still possible for all users.

## Motivation and context

Continue #7127 

## Screenshots / GIFs

image

## Tested devices

- [ ] Physical
- [x] Emulator
- OS version(s):

## Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [x] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

File: vector-app/src/debug/java/im/vector/app/features/debug/features/DebugFeaturesStateFactory.kt
Comment (bmarty) [inline vector-app/src/debug/java/im/vector/app/features/debug/features/DebugFeaturesStateFactory.kt:107]: Adding a lab setting should not make us to remove the feature flag. The feature flag could help to control the display of the lab setting. Not blocking this PR though.

File: vector-config/src/main/res/values-v29/config-settings.xml
Comment (bmarty) [inline vector-config/src/main/res/values-v29/config-settings.xml:11]: That's a nice solution, hence could be hard to understand for fork maintainers.
I would rather test API version in the code, to eventually hide the setting, even if the `bool` is set to `true`.

File: vector-config/src/main/res/values/config-settings.xml
Comment (bmarty) [inline vector-config/src/main/res/values/config-settings.xml:52]: If you keep this, I would rather comment like this:
```suggestion
    <bool name="settings_labs_enable_voice_broadcast_visible">false</bool> <!-- Note: this value must always be false. See in values-v29/config-settings to toggle the visibility of this flag. -->
```

But I would definitely go for the test at runtime, to avoid any mistakes.

So this could be changed to 

 ```suggestion
    <!-- Note: on device with API < 29, setting this value to true has no effect. -->
    <bool name="settings_labs_enable_voice_broadcast_visible">false</bool>
```

**Review by bmarty — APPROVED** [review]
Just some minor remarks on the resources, not really blocking.

File: library/ui-strings/src/main/res/values/strings.xml
Comment (giomfo) [inline library/ui-strings/src/main/res/values/strings.xml:3350]: @Florian14 please replace this wording with "Be able to record and send voice broadcast in room timeline."

File: library/ui-strings/src/main/res/values/strings.xml
Comment (bmarty) [inline library/ui-strings/src/main/res/values/strings.xml:3350]: Looks like there is an extra `‡` at the end of the sentence.

File: changelog.d/7393.wip
Comment (bmarty) [inline changelog.d/7393.wip:1]:
```suggestion
[Voice Broadcast] Enable the feature (behind a lab flag and only for Android 10 and up).
```

**Review by bmarty — COMMENTED** [review]
Some latest remarks

File: library/ui-strings/src/main/res/values/strings.xml
Comment (giomfo) [inline library/ui-strings/src/main/res/values/strings.xml:3349]: @Florian14 feel free to replace the "Voice" with "voice". It would be more adapted to the other flags

**Pull request merged**
